spec/requests/admin/invitations_spec.rb

```diff
@@ -29,6 +29,12 @@
            params: { user: { email: "hey#{rand(1000)}@email.co", name: "Roger #{rand(1000)}" } }
       expect(User.last.registered).to be false
     end
+
+    it "does not create an invitation if a user with that email exists" do
+      post "/admin/invitations",
+           params: { user: { email: admin.email, name: "Roger #{rand(1000)}" } }
+      expect(flash[:error].present?).to be true
```

Comment:
_Non-blocking nitpick_: Maybe we could also test that the user/invitation count does not change here like we do in the [test below](https://github.com/forem/forem/blob/38f294881b9c04ab56d6d8f15e40ef1b32adfc2b/spec/requests/admin/invitations_spec.rb#L47)?

Reply:
Thank you for adding more expectations, @Zhao-Andy! 😊